src/EditorFeatures/Core.Wpf/InlineRename/UI/SmartRename/SmartRenameViewModel.cs

```diff
@@ -64,6 +64,10 @@ internal sealed class SmartRenameViewModel : INotifyPropertyChanged, IDisposable
         }
     }
 
+#pragma warning disable CA1822 // Mark members as static - used in xaml
+    public string GetSuggestionsTooltip => EditorFeaturesWpfResources.Get_AI_suggestions;
```

Comment:
Could we make this static instead of suppress the warning? Maybe directly binded to the static resources in xaml?

Reply:
Good idea!